docs/version0.42.md

```diff
@@ -42,6 +43,16 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 The dump extractor tool, OpenJ9 `jextract`, that was deprecated since the [0.26.0 release](version0.26.md) is now removed from Java 21 and later. The [`jpackcore`](tool_jextract.md) tool replaced the OpenJ9 `jextract` tool after its deprecation. ![End of content that applies to Java 21 (LTS) and later](cr/java_close_lts.png)
 
+### Change in the `System.gc()` call behavior
+
+The `System.gc` call triggers a global garbage collection (GC) cycle, which is a stop-the-world GC. If the `System.gc()` call was made during an active GC cycle where the concurrent marking was in progress, the mark status of objects might not be updated and the now unreachable objects might remain as marked.
```

Comment:
Since there are concurrent GCs that are not just marking, this statement can be generalized. With optionally some more elaborate explanation, something like this, but feel free to re-word:

If the `System.gc()` call was made during an active GC cycle where the concurrent operation was in progress, the liveness status of objects might not be updated and the now unreachable objects that were live at the moment the concurrent operation started might remain live.